src/Compilers/Core/Portable/Symbols/IPropertySymbol.cs

```diff
@@ -110,5 +110,22 @@ public interface IPropertySymbol : ISymbol
         /// The list of custom modifiers, if any, associated with the type of the property. 
         /// </summary>
         ImmutableArray<CustomModifier> TypeCustomModifiers { get; }
+
+        /// <summary>
+        /// If this is a partial property implementation part, returns the corresponding
+        /// definition part.  Otherwise null.
+        /// </summary>
+        IPropertySymbol? PartialDefinitionPart { get; }
```

Comment:
Looking at the usages of the existing public APIs for partial methods, I feel there are more IDE scenario and potentially EnC scenarios that will need to be adjusted for partial properties/indexers. Are those known/understood?

Reply:
IDE scenarios outside of hte min-bar are entirely optional to change.